### DISCUSSION TITLE
Support for 2 character prefixes in autolink references

### DISCUSSION THREAD

**Discussion by recaph** [community/community#23243] [New to GitHub]
<p>We have US &amp; DE as prefixes for stories and defects in our project tracker. Is it possible to relax the requirement to allow 2 character prefix? We are having to prefix another character before the prefixes to make this work.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/recaph">@recaph</a>, Thanks for taking the time to provide this feedback! I’ve taken your suggestion and passed it along to the appropriate teams.</p>
